CMakeLists.txt

```diff
@@ -58,7 +58,8 @@ else()
   set(Protobuf_USE_STATIC_LIBS ON)
 endif()
 
-# Set C++11 as standard for the whole project
+# Required to use /std:c++14 or higher on Windows
```

Comment:
I suggest using C++17 for Windows builds. Then you can use std::string_view in your following code. Every VC++ compiler on Windows supports C++17.